news/2020-03-18-new-featured-artist-leaf.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "New Featured Artist: LeaF"
+date: 2020-03-18 06:00:00 +0000
+---
+
+We're proud to welcome **LeaF** aboard as our latest addition to the Featured Artist roster!
+
+![](https://assets.ppy.sh/artists/73/header.jpg)
+
+Those with particularly calamitous fortunes are likely well acquainted with the wonder that is **LeaF** and his genre-defying rhythmic mastery. From tracks like *Calamity Fortune*, *MEPHISTO*, *Wizdomiot*, *I*, *Evanescent* and more, we've seen a LOT of **LeaF** over the course of osu!'s history, with many of the beatmaps charted to his work serving as cultural icons in their own right.
+
+We've lined up **30** tracks for you to test your mapping mettle with, in a variety of short and extended versions to boot.
+
+In case you're not in the know, check out a sample of some of his work below:
+
+### Aleph-0
+
+Check out this [Loved beatmap](https://osu.ppy.sh/beatmapsets/884669) by [Mr HeliX](https://osu.ppy.sh/users/2330619), and this [popular Loved osu!mania chart](https://osu.ppy.sh/beatmapsets/518951) by [jakads](https://osu.ppy.sh/users/259972)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/73/previews/1980.mp3" type="audio/mpeg">
+</audio>
+
+### Calamity Fortune
+
+Check out this [extremely popular classic beatmap](https://osu.ppy.sh/beatmapsets/65855) hosted by [Flower](https://osu.ppy.sh/users/1033017)!
```

Comment:
Calamity Fortune's beatmap link is incorrect, should link to https://osu.ppy.sh/beatmapsets/96103 instead